docs/001_develop/03_client-capabilities/006_charts/002_g2plot-chart/03_chart_html_api.mdx

```diff
@@ -0,0 +1,430 @@
+---
+title: 'Chart Declerative HTML API'
+sidebar_label: 'Chart Declerative HTML'
+id: chart-html-api
+keywords: [genesis, foundation, ui, chart, g2plot-chart, g2plot]
+tags:
+  - connected
+  - datasource
+  - chart
+  - g2plot-chart
+  - g2plot
+pagination_next: develop/client-capabilities/forms/client-forms
+---
+
+Declarative HTML API enables developers to create and configure charts directly in HTML using simple, semantic tags. It streamlines the charting process by embedding data and configuration options within HTML elements, avoiding complex JavaScript code. 
+
+## G2Plot Chart
+
+Top level web component, which is used to initialise a chart.
+```html
+<rapid-g2plot-chart></rapid-g2plot-chart>
+```
+
+### DOM API
+
+Property and attribute binding examples for Genesis Component syntax. Closing tag omitted.
+
+#### Attributes
+
+<table>
+  <thead>
+    <tr>
+      <th>Attribute</th>
+      <th>Type</th>
+      <th>Use</th>
+      <th>Example</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>type</td>
+      <td><code>[ChartTypes](./docs/api/g2plot-chart.charttypes.md)</code></td>
+      <td>The type to use for the chart. If not set, the default bar type will be used.</td>
+<td>
+
+```typescript
+<rapid-g2plot-chart type="line">
+```
+
+</td>
+    </tr>
+  </tbody>
+</table>
+
+#### Properties
+
+<table>
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Type</th>
+      <th>Use</th>
+      <th>Example</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>config</td>
+      <td><code>[ChartConfig](./docs/api/g2plot-chart.chartconfig.md)</code></td>
+      <td>The configuration to use for the chart.</td>
+<td>
+
+```typescript
+<rapid-g2plot-chart :config=${(x) => x.chartConfig}>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>data</td>
+      <td><code>any[]</code></td>
+      <td>The data to display in the chart.</td>
+<td>
+
+```typescript
+<rapid-g2plot-chart :data=${(x) => x.chartData}>
+```
+
+</td>
+    </tr>
+  </tbody>
+</table>
+
+#### Slots
+
+| Slot | Use |
+|---|---|
```

Comment:
For completeness sake we should add the Default slot here too, and say that is where the chart datasource goes (and you can do an internal page link to that section)